resources/views/layouts/app.blade.php

```diff
@@ -355,12 +355,12 @@
 
         <!-- Javascript Libs -->
         <!-- jQuery 2.1.4 -->
-        <script src="js/plugins/jQuery/jQuery-2.1.4.min.js"></script>
+        <script src="{{ url('js/plugins/jQuery/jQuery-2.1.4.min.js') }}"></script>
         <!-- Bootstrap 3.3.5 -->
-        <script src="js/bootstrap.min.js"></script>
+        <script src="{{ url('js/bootstrap.min.js') }}"></script>
         <!-- FastClick -->
-        <script src="js/plugins/fastclick/fastclick.js"></script>
+        <script src="{{ url('js/plugins/fastclick/fastclick.js') }}"></script>
         <!-- AdminLTE App -->
-        <script src="js/app.min.js"></script>
+        <script src="{{ url('/js/app.min.js') }}"></script>
```

Comment:
Shouldn't this be:
`<script src="{{ url('js/app.min.js') }}"></script>`
instead of:
`<script src="{{ url('/js/app.min.js') }}"></script>`

(Admitted, I have no idea - but it looks like all the lines here should follow the same format.)